src/onboarding/github.md

```diff
@@ -0,0 +1,61 @@
+# GitHub Tickets
+
+GitHub is a service that provides code hosting and versioning. We also use it for its collaborative features, such as issue/bug tracking and project boards. This documents explains how to use GitHub to keep track of tasks.
+
+## Repositories
+
+Here is the homepage for the [toronto-community-network](https://github.com/tomeshnet/toronto-community-network) repository.
+
+![homepage](./images/homepage.png)
+
+A repository (or "repo") is like a project on GitHub, with a collection of files and a name. You can see the name of this one, `toronto-community-network` and that it was created by the [tomeshnet](https://github.com/tomeshnet) organization.
```

Comment:
Maybe "project" would be confused with "github project". `like a project` does not really add anything as project is not well defined.  Also refering to it "on GitHub" adds to the confusion vs "github projects".


Also not sure about the tone. If this is a manual it should not use "you can see" like wording.

```suggestion
A repository (or "repo") is a collection of files, issues, and project boards grouped uner a name. The name of this one is `toronto-community-network` and that it was created by the [tomeshnet](https://github.com/tomeshnet) organization.
```